metricbeat/module/ceph/cluster_disk/cluster_disk_integration_test.go

```diff
@@ -15,20 +15,31 @@
 // specific language governing permissions and limitations
 // under the License.
 
+// +build integration
+
 package cluster_disk
 
 import (
 	"fmt"
 	"os"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+
 	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
 )
 
 func TestData(t *testing.T) {
-	f := mbtest.NewEventFetcher(t, getConfig())
-	err := mbtest.WriteEvent(f, t)
-	if err != nil {
+	t.Skip("Skipping data generation test")
```

Comment:
CI test aren't launched with `-data` option but this is checked in `mbtest.WriteEventsReporterV2` in line 42. An error can still be raised in line 36 if ceph is not started by compose.

The problem is that Ceph start with Compose is buggy and it was omitted consistently in all Metricsets

Reply:
I see. As soon as https://github.com/elastic/beats/pull/10648 is merged we should switch over Ceph to generate the data file based on an http server as the current tests already use a http server.

Let's skip it for now but open a follow up issue to track:

* Implementation of data generation based on https://github.com/elastic/beats/pull/10648
* And investigation in how we can tests ceph against a "real" server like we do for all the other modules.